.github/CODEOWNERS

```diff
@@ -0,0 +1,32 @@
+# This file controls default reviewers for LightGBM code.
+# See https://help.github.com/en/articles/about-code-owners
+# for details
+#
+# Maintainers are encouraged to use their best discretion in
+# setting reviewers on PRs manually, but this file should
+# offer a reasonable automatic best-guess
+
+# main C++ code
```

Comment:
`src/*` ?

Reply:
yes good catch. I will also add **Laurae2** and myself for R-specific files in there